students/aravichander/lesson07/Mailroom_unittest.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+
+from Mailroom_lesson07 import createlist,getlist,sumdonor
+
+class MyFuncTestCase(unittest.TestCase):
+	
+	def test_createlist(self):
+		createlist()
+		output = getlist()
+		self.assertEqual(output["Bill Gates"],[10000,12000]) 
+		self.assertTrue(output["Bill Gates"] == [10000,12000]) 
+		self.assertFalse(output["Bill Gates"] == [10000,12001])
+		with self.assertRaises(KeyError):
+			output["Carrot Top"]
+
+#This was a failed testing scenario as the original function already handles the exception
+	# def test_searchdonors(self):
+	# 	createlist()
+	# 	output = getlist()
+	# 	with self.assertRaises(KeyError):
+	# 		searchdonors("Carrot Top")
+
+	def test_sumdonor(self):
+		createlist()
+		self.assertTrue(sumdonor("Paul Allen") == 1050)
```

Comment:
again -- assertEqual, if you are using unittest.

and no need to test False  -- test for what you want -- if it gives the right result, it gives the right result.